CRM/Core/BAO/Navigation.php

```diff
@@ -578,8 +574,6 @@ public static function processMove($nodeID, $referenceID, $position) {
       $incrementOtherNodes = FALSE;
     }
 
-    $transaction = new CRM_Core_Transaction();
-
     // now update the existing nodes to weight + 1, if required.
     if ($incrementOtherNodes) {
       $query = "UPDATE civicrm_navigation SET weight = weight + 1
```

Comment:
Note this still doesn't call hooks when updating *other* items in the tree. That's fine, but makes https://github.com/civicrm/civicrm-core/pull/27814 still important to merge.

Reply:
Agree, seemed unnecessary to call the hooks for potentially a large number of menu items.